jars/cookies/static/admin/css/base_.css

```diff
@@ -863,7 +863,7 @@ img.bigjar {
 .index-login {
     margin-left: auto;
     margin-right: auto;
-    width: 200px;    
+    width: 220px;    
```

Comment:
To adjust the text to not go to new line.

Without fix it looks like this:
![screenshot from 2015-09-14 11 19 28](https://cloud.githubusercontent.com/assets/4907071/9858008/f75de3ee-5ad2-11e5-8729-1217afe7a0ca.png)

With fix it looks like this:
![screenshot from 2015-09-14 11 18 00](https://cloud.githubusercontent.com/assets/4907071/9857997/d6cd847c-5ad2-11e5-8b04-4d09c1b771a9.png)